### PR TITLE
Feature - Building Bonus Yield AI

### PR DESCRIPTION
**Pull request by 1SDANi**
Hopefully this works

### CODE REVIEW

**Comment by 1SDANi** [comment]
Thank god, the DLL is finally not included in the changes.

File: CvGameCoreDLL/CvCityAI.cpp [hunk 1]
```diff
@@ -4052,6 +4052,12 @@ int CvCityAI::AI_buildingValueThreshold(BuildingTypes eBuilding, int iFocusFlags
 						{
 							iTempValue += ((kBuilding.getBonusYieldModifier(iJ, iI) * getBaseYieldRate((YieldTypes)iI)) / 12);
 						}
+
+						// 1SDAN
+						if (kBuilding.getBonusYieldChange(iJ, iI) > 0)
+						{
+							iTempValue += (kBuilding.getBonusYieldChange(iJ, iI) * countNumBonusPlots() * 6);
```
Comment (dguenms) [inline CvGameCoreDLL/CvCityAI.cpp]: Shouldn't it be `countNumBonusPlots((BonusTypes)iJ)`?

**Review by dguenms — CHANGES_REQUESTED** [review]

File: CvGameCoreDLL/CvCityAI.cpp (hunk 1 above)
Comment (1SDANi) [reply CvGameCoreDLL/CvCityAI.cpp]: oof, my bad. Fixing.

**Review by dguenms — APPROVED** [review]

**Comment by dguenms** [comment]
I merged this manually with the DLL, thanks!

**Pull request closed without merging**